Source/WUtils.swift

```diff
@@ -23,4 +23,47 @@ public enum xAlignment {
 }
 public enum yAlignment {
     case Top, Center, Bottom
-}
\ No newline at end of file
+}
+
+public class WUtils {
```

Comment:
Solid point. As long as we think it will provide value for consumers and there's the chance of them wanting to use it, then let's keep it like this.